src/pages/List.js

```diff
@@ -56,11 +67,59 @@ export default function List({ token }) {
   };
 
   function compareTimeStamps(lastPurchased) {
-    const currentElapsedMilliseconds = Date.now();
-    const millisecondsInOneDay = 86400000;
-    return currentElapsedMilliseconds - lastPurchased < millisecondsInOneDay;
+    // first check to see if lastPurchased === null in database
+    if (lastPurchased === null) {
+      return false;
+    }
+    // use DateTime package to get current time
+    const now = DateTime.now();
+
+    // initialize how many milliseconds there are in a day for calculation
+    const millisecondsInADay = 86400000;
+
+    // convert now to days
+    const nowInDays = Math.floor(now.ts / millisecondsInADay);
+
+    // do the same conversion for last_purchased as lastPurchased
+    const lastPurchasedToDays = Math.floor(
+      DateTime.fromISO(lastPurchased).ts / millisecondsInADay,
+    );
+
+    return nowInDays - lastPurchasedToDays === 0;
```

Comment:
This can be DRYed up along with the other two instances of this code in the component. We can clean this up to not be conducting the same date conversions in three places.

Reply:
Now that we have `latestInterval` working for us, I'm going now attempt to extract that into a helper function I can call in the three separate functions: `markItemPurchased`, `compareTimeStamps` and `checkForInactiveItem`. (wish me luck!)

Reply:
@jamesncox 🍀 🍀

Reply:
GOT IT! Works and looks great IMO! 

@skylerwshaw @yenly ready for ONE FINAL REVIEW (lord wiliing 😂😂😂)